lib/services/android-project-service.ts

```diff
@@ -273,13 +275,13 @@ class AndroidProjectService implements IPlatformProjectService {
 				let libsFolderContents = this.$fs.readDirectory(libsFolderPath).wait();
 				_(libsFolderContents)
 					.filter(libsFolderItem => path.extname(libsFolderItem) === ".jar")
-					.map(jar => this.addLibrary(this.platformData, path.join(libsFolderPath, jar)).wait())
+					.map(jar => this.addLibrary(path.join(libsFolderPath, jar)).wait())
```

Comment:
.map converts (maps) its input values to its output values. In this case, .each is a better choice.